src/mysql_shell/clients/cluster.py

```diff
@@ -3,13 +3,14 @@
 
 import json
 import logging
+from typing import Mapping
```

Comment:
nit: I have a petty habit of doing `import typing as t` so that the members of the module don't pollute the global namespace, what do you think of that? (of course there's consistency considerations, but wanted to know your thoughts still)